.github/ISSUE_TEMPLATE/support-create-a-new-github-team.md

```diff
@@ -0,0 +1,16 @@
+---
+name: 'Support: Create a new GitHub Team'
```

Comment:
Suggestion for Clarification:
- "Create a new GitHub Team in the consortium GitHub organization"

Reply:
adapted your proposals to make it clear where we create things, see https://github.com/eclipse-tractusx/sig-infra/pull/10/commits/54e87d6f9a8ee097475f50af445802f520002e39